packages/disploy/src/structs/managers/ChannelManager.ts

```diff
@@ -14,7 +14,7 @@ export class ChannelManager extends Base {
 	 * @param guildId The ID of the guild to lock the manager to.
 	 */
 	public constructor(app: App, guildId?: string) {
-		super(app);
+		super(app, {});
```

Comment:
In a future PR when `.raw` is implemented, we should make a new class called `BaseDiscordStructure` which would be `Base` in its current state, and `Base` would just have the `App` class attached to it.